.golangci.yml

```diff
@@ -0,0 +1,93 @@
+# This file was inspired by the golangci-lint one:
```

Comment:
we seem to have two approaches to running in CI, one with the Makefile and the other with the Docker image

Reply:
got an example of the other one?

Reply:
https://github.com/ONSdigital/dp-upload-service

Reply:
sure - I guess the advantage of having the install in the makefile means that if you don't have it installed locally it would install. If that's what you're getting at?